client/src/components/AppBar/style.js

```diff
@@ -0,0 +1,73 @@
+import { styled } from '@mui/material/styles';
+import BaseTypography from '@mui/material/Typography';
+import BaseBox from '@mui/material/Box';
+import IconBtn from '@mui/material/IconButton';
+import AppBar from '@mui/material/AppBar';
+import MenuUI from '@mui/material/Menu';
+import ButtonMUI from '@mui/material/Button';
+import { NavLink } from 'react-router-dom';
+
+const WHITE_COLOR_GENERAL = '#fff';
```

Comment:
I suggest to keep colors separately somewhere in a dedicated module. Also use unified approach for color values. I suggest to change '#fff' to '#ffffff'

Reply:
Fixed